packages/FormElement/src/components/Error/Error.js

```diff
@@ -13,22 +13,28 @@ const defaultProps = {
   children: null,
 };
 
-function ErrorMessage(props) {
+function Error(props) {
+  const [ariaErrorId] = React.useState(nanoid);
+  const { addIdToAriaDescribedBy } = React.useContext(FormElementContext);
   const { children, ...moreProps } = props;
 
+  React.useEffect(() => {
+    addIdToAriaDescribedBy({ ariaErrorId });
+  }, []);
+
   if (!children) return null;
 
   return (
-    <div css={errorMessageStyles} data-pka-anchor="form-element.error" {...moreProps}>
+    <div role="alert" css={errorStyles} data-pka-anchor="form-element.error" {...moreProps} id={ariaErrorId}>
```

Comment:
Oh, this didn't have `role=alert` before? Hmm. I noticed in Safari it no longer reads it out, but that was in a static example, not dynamic.  But still, I am a bit worried about JAWS, since this makes it a "live region" and I think with the `<Toast>` we needed to implement a timeout to get it to work properly.  Let's omit it for now but maybe open a GitHub issue, because I think it's semantically correct to make this an `alert`, just think we should carefully test it before making the change.  This is an important component.

Reply:
👌